Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -170,43 +176,57 @@ script:
     - contextPath: File.SSDeep
       description: SSDeep hash of the file.
       type: string
-  - arguments:
-    - description: Commit description.
-      name: description
-    description: Commits a configuration to the Palo Alto firewall or Panorama, but
-      does not validate if the commit was successful. Committing to Panorama does
-      not push the configuration to the firewalls. To push the configuration, run
-      the panorama-push-to-device-group command.
-    name: panorama-commit
+    description: Gets the predefined threats list from a Firewall or Panorama and
+      stores it as a JSON file in the context.
+  - name: panorama-commit
+    arguments:
+    - name: description
+      description: Commit description.
+    - name: use_polling
```

let's call it polling all across the integration. to conform with AFv2 code.
```suggestion
    - name: polling
```